uwb_platform_interface/lib/uwb_platform_interface.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+///Contains all the calls that needs to be implemented in the platform specific plugins
+abstract class UwbPlatform extends PlatformInterface {
+  /// Constructs a UwbPlatform.
```

Comment:
Make the `UwbPlatform` a documentation link so that is will standout more clearly in the generated documentation (reasoning is explained [here](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments)):

```suggestion
  /// Constructs a [UwbPlatform].
```